mnemonic-hadoop/mnemonic-hadoop-mapreduce/src/test/java/org/apache/mnemonic/mapred/MneMapredBufferDataTest.java

```diff
@@ -74,7 +74,7 @@
   @BeforeClass
   public void setUp() throws IOException {
     m_workdir = new Path(
-        System.getProperty("test.tmp.dir", DEFAULT_WORK_DIR));
+            System.getProperty("test.tmp.dir", DEFAULT_WORK_DIR));
```

Comment:
looks the IDE adds those extra white-spaces, could you please remove that ? and there are more lines below, thanks.

Reply:
+1, I will fix those.

Reply:
@bigdata-memory Addressed the requested changes. Please check.